#TODO Implement: db_obj.load("UNKNOWN_ID") should raise error